R/wwinference.R

```diff
@@ -87,22 +84,25 @@ wwinference <- function(ww_data,
     include_ww = as.numeric(model_spec$include_ww),
     compute_likelihood = 1
   )
+  # We want to return these to the user
+  input_data <- input_data_and_args$input_data
+  stan_args <- input_data_and_args$stan_args
```

Comment:
Did tests pass here? I am concerned that as written it will no longer pass the ww and count data (which iirc are under `input_data`, not `stan_args`, to the `sample()` call. Am I missing something? If I am not and this is a potential bug, perhaps worth writing a test that would've caught this.

Reply:
Yes they passed, this is redundant bc `stan_args` contains the full set of data needed to be passed to stan. The `input_data` is just a nested list of the data that was passed to stan with dates bound to it. I wanted to make it a list so that as we increase the input data (e.g. and an adjacency matrix) this becomes easier to modify because the set of outputs from `wwinference()` will be stable. See #49 for this, sorry there is a bit of scope creep in this PR since I wanted to write the tests of `get_stan_data()` with the correct outputs